File: Programming Basics with Python/First Steps in Coding/Exercise/08_basketball_equipment.py
annual_fee = int(input())

snickers = annual_fee - annual_fee * 0.4
team_uniform = snickers - snickers * 0.2
ball = team_uniform / 4
accessories = ball / 5

total_sum = annual_fee + snickers + team_uniform + ball + accessories

print(total_sum)

File: Basics with JavaScript/01. First Steps In Coding/Exercise/08. basketballEquipment.js
function basketball(input) {
    let annualTax = Number(input[0]);
    let snickers = annualTax * 0.6;
    let clothes = snickers * 0.8;
    let ball = clothes / 4;
    let accessories = ball / 5;
    let total = annualTax + snickers + clothes + ball + accessories;
    console.log(total);
}

basketball(["365 "]);